src/labels/CustomLabels.labels

```diff
@@ -2752,6 +2752,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>- Household Duplicate</value>
     </labels>
     <labels>
+        <fullName>hhErrorInvalidContactFieldName</fullName>
+        <categories>Households, Household Naming</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>hhErrorInvalidContactFieldName</shortDescription>
+        <value>The field, {0}, does not match any field name on Contact.</value>
```

Comment:
@smarovich , please ignore request for the error message review. The message is not required any longer. The changes will be done in the new commit.